identity-service/src/main/java/com/softserve/identityservice/model/UserInfoResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.softserve.identityservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
can you remind me, why don't you use @data insted all of these annotations?

Reply:
I think it's more clearly.